### PR TITLE
Jm/ftlos

### PR DESCRIPTION
**Pull request by jawwadmukhtargmail**

### CODE REVIEW

File: GroupService/GroupService.Repo/ApplicationDbContext.cs [hunk 1]
```diff
@@ -95,6 +97,48 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+
+                entity.ToTable("RequestHelpJourney", "Website");
+
+                entity.Property(e => e.GroupId).HasColumnName("GroupID");
+
+                entity.Property(e => e.Source)
+                    .HasMaxLength(100)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.RequestHelpSourceId).HasColumnName("RequestHelpSourceID");
```
Comment (mike-f50) [inline GroupService/GroupService.Repo/ApplicationDbContext.cs:134]: Could we go with RequestHelpFormVariant?  I'm trying to get away from Source.

File: GroupService/GroupService.Repo/ApplicationDbContext.cs [hunk 2]
```diff
@@ -95,6 +97,48 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
...
+
+                entity.ToTable("RegistrationJourney", "Website");
+
+                entity.Property(e => e.GroupId).HasColumnName("GroupID");
+
+                entity.Property(e => e.Source)
+                    .HasMaxLength(100)
+                    .IsUnicode(false);
+
+                entity.Property(e => e.RegistrationSourceId).HasColumnName("RegistrationSourceID");
```
Comment (mike-f50) [inline GroupService/GroupService.Repo/ApplicationDbContext.cs:113]: Could we go with RegistrationFormVariant?

File: GroupService/GroupService.Repo/EntityFramework/Entities/RegistrationJourney.cs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace GroupService.Repo.EntityFramework.Entities
+{
+    public class RegistrationJourney
+    {
+        public int GroupId { get; set; }
+        public string Source { get; set; }
+        public byte RegistrationSourceId { get; set; }
```
Comment (mike-f50) [inline GroupService/GroupService.Repo/EntityFramework/Entities/RegistrationJourney.cs:11]: again...

File: GroupService/GroupService.Repo/EntityFramework/Entities/RequestHelpJourney.cs [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+using System.Collections.Generic;
+using System.Text;
+
+namespace GroupService.Repo.EntityFramework.Entities
+{
+    public class RequestHelpJourney
+    {
+        public int GroupId { get; set; }
+        public string Source { get; set; }
+        public byte RequestHelpSourceId { get; set; }
```
Comment (mike-f50) [inline GroupService/GroupService.Repo/EntityFramework/Entities/RequestHelpJourney.cs:11]: and again :)

File: GroupService/GroupService.Repo/ApplicationDbContext.cs (hunk 1 above)
Comment (jawwadmukhtargmail) [reply GroupService/GroupService.Repo/ApplicationDbContext.cs:134]: done

File: GroupService/GroupService.Repo/EntityFramework/Entities/RegistrationJourney.cs (hunk 3 above)
Comment (jawwadmukhtargmail) [reply GroupService/GroupService.Repo/EntityFramework/Entities/RegistrationJourney.cs:11]: done

File: GroupService/GroupService.Repo/EntityFramework/Entities/RequestHelpJourney.cs (hunk 4 above)
Comment (jawwadmukhtargmail) [reply GroupService/GroupService.Repo/EntityFramework/Entities/RequestHelpJourney.cs:11]: done

File: GroupService/GroupService.Repo/ApplicationDbContext.cs (hunk 2 above)
Comment (jawwadmukhtargmail) [reply GroupService/GroupService.Repo/ApplicationDbContext.cs:113]: done

**Pull request merged**